src/Sulu/Bundle/AdminBundle/Resources/js/services/ResourceRequester/ResourceRequester.js

```diff
@@ -10,6 +10,16 @@ const listDefaults = {
 };
 
 export default class ResourceRequester {
+    static get(resourceKey: string, id: number | string) {
+        const baseUrl = resourceMetadataStore.getBaseUrl(resourceKey);
+        return Requester.get(baseUrl + '/' + id);
+    }
+
+    static put(resourceKey: string, id: number | string, data: Object) {
+        const baseUrl = resourceMetadataStore.getBaseUrl(resourceKey);
+        return Requester.put(baseUrl + '/' + id, data);
```

Comment:
This does currently not really work... The thing is that our API relies on getting the `language` attribute passed here (at least it's like that for the snippets). We have to think about how to solve this. I think a real first step would be to sanitize all the query parameters, and name them the same way, not sometimes `locale` and sometimes `language`. The next major release would be a good start for this anyway.